### PR TITLE
Added Ariz button

### LINKED ISSUE #475: "Open Ariz Portfolio" on project page
Button should be added to project page/project overview

Only visible to project owners

Click to navigate to https://arizportfolio.near.page/

Get logos and color pallette from here: https://github.com/arizas/Ariz-Portfolio/tree/main/Logo

### PR DESCRIPTION
**Pull request by Jikugodwill**
- Resolves #475
I've place this next to the hyperlinks as it makes a better position IMO. What do y'all think?

https://github.com/user-attachments/assets/358eb76a-9d68-4279-a642-dc266f559af5

### CODE REVIEW

File: apps/new/widget/page/project/Metadata.jsx [hunk 1]
```diff
@@ -94,13 +94,36 @@ const EditButton = ({ item }) => {
   );
 };
 
+const HyperLink = styled.a`
```
Comment (elliotBraem) [inline apps/new/widget/page/project/Metadata.jsx:97]: Do we have an existing component like this already?

Reminds me of button outline

File: apps/new/widget/page/project/Metadata.jsx [hunk 2]
```diff
@@ -140,13 +163,25 @@ const Metadata = ({
...
+                    profile,
+                  }}
+                />
+                {isOwner && (
+                  <HyperLink
+                    href="https://arizportfolio.near.page/"
+                    className="btn btn-outline"
+                    target="_blank"
+                  >
+                    {/* <img src="https://github.com/arizas/Ariz-Portfolio/blob/30ea2c2451b733aef3ce191f5d5272cf68e83fa8/Logo/Ariz-logo_svart-skrift.png" /> */}
```
Comment (elliotBraem) [inline apps/new/widget/page/project/Metadata.jsx:180]: Doesn't look good with logo?

**Comment by elliotBraem** [comment]
Also, I think it looks decent where you put it, although it might be nice if in the top right (but if you're not convinced, no problem, leave it where it is)

Also, add tests

"should show ariz portfolio button when owner of project"
"should navigate to ariz when clicked"

File: apps/new/widget/page/project/Metadata.jsx (hunk 2 above)
Comment (Jikugodwill) [reply apps/new/widget/page/project/Metadata.jsx:180]: the logo didn't load properly

File: apps/new/widget/page/project/Metadata.jsx (hunk 1 above)
Comment (Jikugodwill) [reply apps/new/widget/page/project/Metadata.jsx:97]: You're right actually, for a second there I thought we couldn't edit the colors on the "Button" 😅

File: aliases.mainnet.json [hunk 3]
```diff
@@ -6,6 +6,5 @@
   "gateway_url": "https://nearbuilders.org",
   "new": "builddao.near",
   "old": "buildhub.near",
-  "potlock": "potlock.near",
-  "every": "every.near"
+  "potlock": "potlock.near"
```
Comment (Megha-Dev-19) [inline aliases.mainnet.json:9]: these changes are already on develop if I remember correctly?

**Pull request merged**